src/DetailsView/components/export-dialog.tsx

```diff
@@ -24,6 +24,7 @@ export interface ExportDialogProps {
     reportExportFormat: ReportExportFormat;
     onExportClick: () => void;
     featureFlagStoreData: FeatureFlagStoreData;
+    afterDismissed?: () => void;
```

Comment:
How come this is optional?

Reply:
This is only used for setting focus after the dialog is closed. In the old ReportExportComponent (which is still used in electron) we don't do any work after dismissing the dialog. Updating state to reflect that the dialog has closed is handled in onClosed, which runs just before before the dialog closes, but since the buttons are not focusable before the dialog closes we can't use onClose to set focus.